NEW_README.md

```diff
@@ -0,0 +1,37 @@
+# MarkLogic Kubernetes Helm Chart
+
+## Prerequisites
+
+To install this chart, you need to install [Helm](https://helm.sh/docs/intro/install/) and [Kubectl](https://kubernetes.io/docs/tasks/tools/).
+
+To set up a Kubernetes Cluster for Production Workload, we recomend using EKS platform on AWS. To bring up a kubernetees cluster on EKS, you can install [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) tool. Please refer to [Using eksctl to Provision MarkLogic Kubernetes Cluster on EKS](README.md/) for detailed steps.
```

Comment:
The link to EKS will need to change

Reply:
Yes, all the links in the readme will point to MarkLogic Kubernetes Documentations and other docs respectively.